### PR TITLE
Refactor `AvatarWrapper` component

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Remove duplicate code by merging two components in a single
* Cover component with tests for #9202 
image

### 🖼️ Screenshots

Right sidebar | Breakout rooms :warning: 
---|---
image | image
New conversation (2) | Poll results
image | image

### 🚧 Tasks

- [ ] Code review

### 🏁 Checklist

- [ ] ⛑️ Tests (unit and/or integration) are included or not required
- [ ] 📘 API documentation in `docs/` has been updated or is not required
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required
- [ ] 🔖 Capability is added or not needed

### CODE REVIEW

File: src/components/AvatarWrapper/AvatarWrapper.vue [hunk 1]
```diff
@@ -84,10 +92,9 @@ export default {
...
 			default: false,
 		},
-
 		showUserStatus: {
 			type: Boolean,
-			default: true,
+			default: false,
 		},
 		showUserStatusCompact: {
 			type: Boolean,
```
[the remark this answers is not in the document]
Comment (ShGKme) [reply src/components/AvatarWrapper/AvatarWrapper.vue:100]: The original `NvAvatar` has this prop `true` by default, but it also has related `showUserStatus` true.

File: src/components/AvatarWrapper/AvatarWrapper.spec.js [hunk 2]
```diff
@@ -1,76 +1,156 @@
...
+				},
+			})
+
+			const avatar = wrapper.findComponent(NcAvatar)
+			await avatar.vm.$nextTick()
+
+			expect(avatar.attributes('title')).toBe(USER_NAME)
+			expect(avatar.attributes('aria-label')).toContain(USER_NAME)
+			const status = wrapper.findComponent('.avatardiv__user-status--online')
+			expect(status.exists()).toBeTruthy()
```
Comment (ShGKme) [inline src/components/AvatarWrapper/AvatarWrapper.spec.js:80]: Setting attributes on the root element of `NcAvatar` is not a part of `AwatarWrapper` component. It's 3rdparty `NcAvatar` internal implementation.

We should test only that name is passed to NcAvatar. All the other is the NcAvatars's job and should be tested in NcAvatar.

`avatardiv__user-status--online` is also a very internal implementation of not a Talk component.

File: src/components/AvatarWrapper/AvatarWrapper.vue [hunk 3]
```diff
@@ -97,16 +104,18 @@ export default {
 			type: Object,
 			default: undefined,
 		},
-
-		menuContainer: {
+		container: {
 			type: String,
 			default: undefined,
 		},
```
[the remark this answers is not in the document]
Comment (Antreesy) [reply src/components/AvatarWrapper/AvatarWrapper.vue:110]: It was used only in on implementation, and same getter was passed as a prop, so i removed it as redundant now

File: src/components/AvatarWrapper/AvatarWrapper.spec.js (hunk 2 above)
Comment (Antreesy) [reply src/components/AvatarWrapper/AvatarWrapper.spec.js:80]: replaced with prop checking

**Pull request merged**